polytope/polytope.py

```diff
@@ -291,26 +291,34 @@ def intersect(self, other, abs_tol=ABS_TOL):
         return reduce(Polytope(iA, ib), abs_tol=abs_tol)
 
     def translation(self, d):
-        """Translate the Polytope by a given vector.
+        """Returns a copy of C{self} translated by the vector C{d}.
 
         Consult L{polytope.polytope._translate} for implementation details.
 
         @type d: 1d array
-        @param d: The translation vector.
-
-        @rtype: L{Polytope}
-        @return: A translated copy of the Polytope.
         """
         newpoly = self.copy()
         _translate(newpoly, d)
         return newpoly
 
     def rotation(self, u, v, theta=None):
-        """Rotate the Polytope. Only simple rotations are implemented at this
+        """Returns a rotated copy of C{self}.
+
+        Use one of two methods that to describe the plane of rotation and the
+        angle of rotation (in radians) with u, v, and theta.
+
+        (1) u and v are the indicies (0, N] of two basis vectors, and theta is
```

Comment:
What I had in mind was raising an error if either `u` or `v` isn't a basis vector. If both are basis vectors, then convert them to indices, and proceed with the rest of the computation as currently done. Your current decision yields a simpler interface, and simple is better than complex (PEP 20).